int32 speed_limit_state
float64 speed_limit_value
---
int32 result

int32 SPEED_LIMIT_ON=1
int32 SPEED_LIMIT_OFF=0